tests/playwright/src/model/pages/chat-page.ts

```
@@ -174,15 +177,15 @@ export class ChatPage extends BasePage {
     await handleDialogIfPresent(this.page);
   }
 
-  async waitForChatHistoryCount(expectedCount: number, timeout = 10_000): Promise<void> {
+  async waitForChatHistoryCount(expectedCount: number, timeout: number = 10_000): Promise<void> {
```

Biome (2.4.7)

[error] This type annotation is trivially inferred from its initialization.

(lint/style/noInferrableTypes)